testsuite/config/cucumber.yml

```diff
@@ -35,7 +35,7 @@ subscription_matching: --tags @scope_subscription_matching
 sumatoolbox: --tags @scope_sumatoolbox
 tomcat: --tags @scope_tomcat
 traditional_client: --tags @scope_traditional_client
-ubuntu: --tags @scope_ubuntu
+ubuntu: --tags @scope_deblike
```

Comment:
Just a reminder to not forget to change the scopes on the CI files themselves too

Reply:
As said in this PR's description, it is addressed in:

SUSE/susemanager-ci#595

Reply:
I missed it in the description, sorry :see_no_evil: